### PR TITLE
Update app-governance-get-started.md

### PR DESCRIPTION
**Pull request by AndrewVahidnia**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@AndrewVahidnia : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: CloudAppSecurityDocs/app-governance-get-started.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ You can purchase app governance as an add-on license to any license that entitle
 
 For a list of these licenses, see the [Defender for Cloud Apps licensing datasheet](https://aka.ms/mcaslicensing). You can confirm the licenses in your tenant at Microsoft 365 admin center.
 
+App governance is currently not available as an addon for Microsoft Defender for Cloud Apps Discovery (DCAS), and Microsoft Office 365 Cloud App Security (OCAS).
```
Comment (dcurwin) [inline CloudAppSecurityDocs/app-governance-get-started.md:30]:
```suggestion
App governance is currently not available as an add-on for Microsoft Defender for Cloud Apps Discovery (DCAS), and Microsoft Office 365 Cloud App Security (OCAS).
```

**Comment by PRMerger13** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: CloudAppSecurityDocs/app-governance-get-started.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ You can purchase app governance as an add-on license to any license that entitle
 
 For a list of these licenses, see the [Defender for Cloud Apps licensing datasheet](https://aka.ms/mcaslicensing). You can confirm the licenses in your tenant at Microsoft 365 admin center.
 
+App governance is currently not available as an add-on for Microsoft Defender for Cloud Apps Discovery (DCAS), and Microsoft Office 365 Cloud App Security (OCAS).
```
Comment (dcurwin) [inline CloudAppSecurityDocs/app-governance-get-started.md:30]: This is not a familiar term: Microsoft Defender for Cloud Apps Discovery (DCAS), and certainly the acronym is not used.

Also, I don't think we should use the OCAS acronym as well.

**Review by dcurwin — COMMENTED** [review]
Please address the comment before publishing,

**Comment by Court72** [comment]
@dcurwin 

Thanks for reviewing the proposed changes. *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

**Pull request closed without merging**